### ISSUE TITLE
uniformConverter.sqf

### ISSUE THREAD

**Issue by WhatsUpDoc127** [A3Wasteland/ArmA3_Wasteland.Altis#458]
If an Independent player purchases a Stealth Uniform from the General Store then places that purchased uniform on the ground or in a box/crate, the independent player cannot put that uniform back on again.

I understand uniformConverter.sqf is suppose to fix this issue and it works on initial purchase but if independent wants to purchase multiple uniforms and place them in a box they cannot wear them again once taken off. 

I know BIS limited independent uniforms severely and hope this can be fixed.

**Comment by AgentRev** [comment]
This cannot be fixed, especially if we're talking about putting them in boxes. It's an engine limitation. Any sort of wretched workaround I can possibly think of would cause duping exploits. BIS were too lazy to add an indie thermal suit, and too dumb to let people take enemy uniforms. They were told many times that this was very inconvenient but they keep turning a deaf ear.

**Comment by AgentRev** [comment]
If they would actually respond, it would probably be total bullshit in the likes of "the thermal suits were made specifically for Tanoa, and the AAF faction is not supposed to be on Tanoa, therefore they don't need a thermal suit, hurr durr!"

**Comment by codexGW** [comment]
Lol

**Comment by WhatsUpDoc127** [comment]
LOL I understand AgenRev and I figure there was a limitation, mainly after reading your uniformConverter.sqf. 

I thought it was an inventory issue but I guess not...Oh well hopefully the upcoming executables will fix these issues...Okay that was a joke!!!

I just don't understand BIS sometimes. But hey if somebody wants the thermals then they can join Opfor!!!

**Comment by nerdalertdk** [comment]
Yeah always join opfor, they get buildtin armor in the uniforms also gg

**Comment by AgentRev** [comment]
only ghillie suits have armor for Opfor, I switched the default uniforms to unarmored ones earlier this summer
